recipes/pagerduty.rb

```diff
@@ -27,7 +27,7 @@ def using_old_pagerduty_key_attribute?
 if using_old_pagerduty_key_attribute?
   Chef::Log.warn('The nagios.pagerduty_key attribute is deprecated. It is replaced by the nagios.pagerduty.key attribute.')
   Chef::Log.warn('Assigning nagios.pagerduty.key from nagios.pagerduty_key now.')
-  node.normal['nagios']['pagerduty']['key'] = node['nagios']['pagerduty_key']
+  node.default['nagios']['pagerduty']['key'] = node['nagios']['pagerduty_key']
```

Comment:
We probably have to keep the node.normal here since switching it to default breaks the ability to change the one that's already set on the node.

Reply:
Then I guess we change to override, just trying to sort out the depreactions